index.bs

```diff
@@ -5521,8 +5523,15 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
     let |currentBe| and |currentBs| be the values of the [=authData/flags/BE=] and [=authData/flags/BS=] bits, respectively,
     of the <code>[=flags=]</code> in |authData|.
     Compare |currentBe| and |currentBs| with
-    <code>|credentialRecord|.[$credential record/backupEligible$]</code> and <code>|credentialRecord|.[$credential record/backupState$]</code>
-    and apply [=[RP]=] policy, if any.
+    <code>|credentialRecord|.[$credential record/backupEligible$]</code> and <code>|credentialRecord|.[$credential record/backupState$]</code>:
+
+    1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is set, verify that |currentBe| is set.
+
+    1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is not set, verify that |currentBe| is not set.
+
+    1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is not set, verify that |currentBs| is not set.
+
+    1. Apply [=[RP]=] policy, if any.
```

Comment:
The new step during registration is always in force, but these criteria are only enforced "if the credential backup state is used as part of Relying Party business logic or policy". We should have the same enforcement criteria in both cases: either always, or only "if the credential backup state is used as part of Relying Party business logic or policy".

I would favour always rejecting `BE=0; BS=1`, but the comparison with `|credentialRecord|.[$credential record/backupEligible$]` probably needs to stay within the qualified case since it's optional for the RP to store that value.

Reply:
I tried to address this in https://github.com/w3c/webauthn/pull/1907/commits/3e0395f5d5cdb6cac5b814ef9492be2f803f8897, basically copy-pasted your suggested wording in the comment above to above these conditional checks based on `credentialRecord`. That should enforce the "impossibility" of an authenticator returning `be:0,bs:1` too.

Reply:
Looks good! The third conditional check can be eliminated now, see new thread.